### ISSUE TITLE
Pass CLI arguments to cypress.run 

### ISSUE THREAD

**Issue by ArturT** [KnapsackPro/knapsack-pro-cypress#10]
If you would like to pass argument like `--record` to cypress then normally you would do:

```
npx cypress run --record
```

At the moment we cannot do the same with `@knapsack-pro/cypress` as it is not passing the CLI argument to [`cypress.run` (see source code of our lib)](https://github.com/KnapsackPro/knapsack-pro-cypress/blob/9ae94d899d6d766c1dc96cd63a1ab38739056ee2/src/knapsack-pro-cypress.ts#L29).

```
# --record will be ignored
$(npm bin)/knapsack-pro-cypress --record
```

Cypress docs says that `cypress.run` accepts the arguments:
https://docs.cypress.io/guides/guides/module-api.html#cypress-run

Note some of them are camelCase like `ciBuildId` but CLI arguments are in format `--ci-build-id`:
https://docs.cypress.io/guides/guides/command-line.html#cypress-run

# TODO
* check if there is a way to pass CLI arguments down to [`cypress.run` function](https://docs.cypress.io/guides/guides/module-api.html#cypress-run).
* alternative solution could be to create ENV variables for each argument and then if `KNAPSACK_PRO_CYPRESS_RECORD=true` then set `record: true` in `cypress.run` function etc. This might be silly idea I guess.

# Side problem
If you would use `record: true` argument in `cypress.run` function then batch of tests fetched from Knapsack Pro API Queue is executed and shown on Cypress Dashboard as single CI build.
So when you consume Queue then multiple CI builds are shown in Cypress Dashboard for each batch of tests fetched from the Queue.

I tried to group the batches with `ciBuildId` that requires to use `group` argument at the same time.

```js
  const { runs: tests, totalFailed } = await cypress.run({
    spec: testFilePaths,
    record: true,
    ciBuildId: 'some-build-id-here',
    group: '@knapsack-pro/cypress',
  });
```

But this does not work as paid plan is required on Cypress to use grouping. We could add this to FAQ once the root issue with passing CLI arguments is solved.

**Comment by ArturT** [comment]
It looks like we could use [minimist](https://www.npmjs.com/package/minimist) lib to parse arguments from `process.argv`.

More about parsing CLI arguments: https://stackabuse.com/command-line-arguments-in-node-js/

`minimist` is already used in Cypress https://github.com/cypress-io/cypress/blob/55b352a11d0a664517ae054c67432d98abc2389e/scripts/run.js#L3

```js
const args = minimist(process.argv.slice(2))
```

**Comment by ArturT** [comment]
This was fixed in PR https://github.com/KnapsackPro/knapsack-pro-cypress/pull/11 and I've released `@knapsack-pro/cypress` [1.1.0](https://github.com/KnapsackPro/knapsack-pro-cypress/blob/master/CHANGELOG.md#v110-2019-03-07).

Now you can pass arguments directly to `@knapsack-pro/cypress` and it will pass it down to `cypress.run`. Here is FAQ example: https://github.com/KnapsackPro/knapsack-pro-cypress#how-to-record-ci-builds-in-cypress-dashboard
